x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/import_view/import_view.js

```diff
@@ -206,7 +207,10 @@ export class ImportView extends Component {
                 // mappings, use this field as the time field.
                 // This relies on the field being populated by
                 // the ingest pipeline on ingest
-                if (mappings.properties[DEFAULT_TIME_FIELD] !== undefined) {
+                if (
+                  isPopulatedObject(mappings.properties) &&
```

Comment:
FYI `isPopulatedObject` can check if certain attributes are present like this:

```
isPopulatedObject(mappings.properties, [DEFAULT_TIME_FIELD])
```

Reply:
Updated in https://github.com/elastic/kibana/pull/146109/commits/82ec821fa4b5bc78be40aa4ccd89a46bf28cd2f4